package.json

```diff
@@ -21,7 +21,7 @@
   "author": "Simon Ihmig <ihmig@kaliber5.de>",
   "license": "MIT",
   "devDependencies": {
-    "bootstrap": "4.0.0-alpha.6",
+    "bootstrap": "^4.0.0-beta",
```

Comment:
FYI you'll also need to tweak index.js the same way: https://github.com/kaliber5/ember-bootstrap/blob/da0b49cda770cda80c3e0066601bf1fedba638e9/blueprints/ember-bootstrap/index.js

Reply:
@bgentry thanks for the reminder, fixed this!